index.html

```diff
@@ -37,8 +38,200 @@ <h1>I'm Joseph Glad to meet you!</h1>
             <li> <a href=""><img src="Images/Vector.svg" alt=""></a></li>
         </ul>
     </main>
+    <section id="secondPage">
```

Comment:
- [ ] Kindly consider applying `display: grid` here in this container instead of `flex` to meet the project requirement.